### ISSUE TITLE
Time Window fields on New/Edit Skill dialog need to include maximum values for screenreader

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1199]
The Time Window input fields need to indicate the maximum values for screenreader users via aria-label or other attributes. Currently the only indication of maximum values are validation failures (which also aren't visible or notified to screenreader users).

**Comment by rmmayo** [comment]
PR: #1428

**Comment by mosefer** [comment]
:+1:
